### ISSUE TITLE
Flatpak installation is taking up an entire 10.5 GB of disk space, is this normal?

### ISSUE THREAD

**Issue by TheBlueQuasar** [jasp-stats/jasp-issues#1909]
### JASP Version

0.16.4

### JASP Module

Unrelated

### What OS are you seeing the problem on?

Flatpak, Ubuntu

### Bug Description

Hello there,
I've installed the flatpak version on my Ubuntu machine (22.04 LTS if it's relevant) and it's taking up 10.5 GB of space.
Is this normal? Or is some kind of bug causing this?
I found it odd as the downloaded data upon the installation was around 2 GB. Pardon me if this is the expected behavior, and just close this issue afterwards.
Attached a [screenshot](https://imgur.com/a/Qx5fBCm) of the Flatpak info command showing the disk space used.

![image](https://user-images.githubusercontent.com/9962568/202840742-03780447-6e38-458f-8634-a1a9beb2d313.png)

### Expected Behaviour

As the downloaded data when installing was 2 GB, most likely, the installation should not take this much space (10.5 GB)

### Steps to Reproduce

1. On a Linux, and preferably Ubuntu 22.04 machine, install JASP
2. Once installed, run the Flatpak command to get the information on the installation: `flatpak info org.jaspstats.JASP` 
3. If the installation is taking up too much space (like > 10 GB), something might be wrong

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by JorisGoosen** [comment]
It sure doesn't sound normal!

It should be way less...
Ill go check my linux systems tomorrow

**Comment by JoKeyser** [comment]
What's (mildly) confusing to me is that the download for the upgrade was much faster than anticipated, if it really were >10 GB large.
So I would not exclude some kind of bug/misunderstanding in flatpak's size display... not sure how to verify the info with the size on disk.

The core files seem reasonably smaller, but maybe it pulled some dependencies elsewhere?

```
$ du -hs /var/lib/flatpak/app/org.jaspstats.JASP
3,3G	/var/lib/flatpak/app/org.jaspstats.JASP
```

**Comment by JorisGoosen** [comment]
You could try something like this: https://www.linuxuprising.com/2019/02/how-to-remove-unused-flatpak-runtimes.html

**Comment by JorisGoosen** [comment]
As far as checking the actual disk size, Ill have to look into that. There must be some way

**Comment by JoKeyser** [comment]
I tried `$ flatpak uninstall --unused`, but it only deleted some stuff from other flatpaks, didn't change the info about JASP.

**Comment by JorisGoosen** [comment]
Yeah and Im installing from scratch now and it does look like a 5.4gb download at least

**Comment by JorisGoosen** [comment]
So looking inside the jasp flatpak (starting as `flatpak run --command=sh org.jaspstats.JASP`) I can see that `/app` at least comes close to what id expect:
```
[📦 org.jaspstats.JASP app]$ du -sh *
12M     bin
4.0K    cleanup-BaseApp.sh
8.0K    etc
2.5M    include
202M    lib
1.8G    lib64
8.7M    libexec
20K     manifest-base-1.json
8.0K    manifest.json
88K     mkspecs
44K     modules
1.2G    Modules
136K    plugins
984K    qml
2.7M    qtwebengine_dictionaries
5.3M    resources
114M    Resources
220K    share
21M     translations
[📦 org.jaspstats.JASP app]$ cd ..
[📦 org.jaspstats.JASP /]$ du -sh app
3.3G    app
```

That also includes some auxilary stuff btw. And comes nowhere near 10.5GB...

**Comment by JoKeyser** [comment]
Regarding the download size, it should be expected that the compressed download is smaller than the actual installed size, [according to this comment](https://github.com/flatpak/flatpak/issues/977#issuecomment-324259556).

The expected download size is 5.4 GB:

```
$ flatpak --system remote-info flathub org.jaspstats.JASP

        ID: org.jaspstats.JASP
       Ref: app/org.jaspstats.JASP/x86_64/stable
      Arch: x86_64
    Branch: stable
Collection: org.flathub.Stable
  Download: 5,4 GB
 Installed: 10,5 GB
   Runtime: org.kde.Platform/x86_64/6.2
       Sdk: org.kde.Sdk/x86_64/6.2
[...]
```

The installed size discrepancy is still a mystery to me.

**Comment by JorisGoosen** [comment]
Yeah but in the end it only downloaded 1.3GB, I misread it earlier. 
Still reads as 10.5GB in `flatpak info`...

I know that flatpak stores the downloaded executables/apps as an [ostree](https://ostree.readthedocs.io/en/latest/) somewhere, but as that is basically a git repo Im not sure how to assess whether the 10.5GB come from there.

Maybe I should ask the people over at flatpak/flathub if they have a clue.

**Comment by TheBlueQuasar** [comment]
This bug seems to have been fixed for me after the 0.17.0 version update. Here's the output for the `flatpak info` command:

`flatpak info org.jaspstats.JASP

JASP - Standard statistical analyses, both classical and Bayesian

          ID: org.jaspstats.JASP
         Ref: app/org.jaspstats.JASP/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 0.17.0
     License: AGPL-3.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 2.4 GB
     Runtime: org.kde.Platform/x86_64/6.3
         Sdk: org.kde.Sdk/x86_64/6.3

      Commit: 008dfb5faba99b4c725653f0fd30a0f9978b400b018584d5155720e1e9ac7cbf
      Parent: 81815cb491003636c4bebe9f16b08a54d20cc502579bd94f26e20fb34c82fcc5
     Subject: v0.17.0 (1c60ec4f)
        Date: 2023-01-30 12:34:07 +0000`

**Comment by JorisGoosen** [comment]
I dont know what went wrong but good it got resolved. If @RensDofferhoff can check with the new beta for 0.17.1 this is still the case we can just close this then.

**Comment by RensDofferhoff** [comment]
Fixed in 0.17.1 and beyond
